mtools/mlaunch/mlaunch.py

```diff
@@ -51,7 +51,6 @@ class MongoConnection(Connection):
     """
```

Comment:
We can remove directConnection from the doc string now.

Reply:
Done